src/app/containers/MediaPlayer/Metadata.jsx

```diff
@@ -14,4 +15,13 @@ const Metadata = aresMediaBlock => {
   );
 };
 
+Metadata.propTypes = {
+  aresMediaBlock: shape({
+    id: string,
+    type: string,
+    model: shape({}),
+    position: array,
```

Comment:
Is it necessary to check the types of `id`, `type`, and `position`, when they're not being used?

Reply:
@simonsinclair they make up the shape of the `aresMediaBlock`, we can't leave it as `any`

Reply:
I understand, but why not just check the property that's being used?
```
aresMediaBlock: shape({
  model: shape({}),
}).isRequired,
```